package.json

```diff
@@ -50,6 +50,7 @@
     "@babel/plugin-transform-runtime": "~7.4.4",
     "@magento/eslint-config": "~1.5.0",
     "@types/jest": "~25.2.1",
+    "babel-plugin-dynamic-import-node": "~2.2.0",
```

Comment:
Another dependency that was being hoisted before that no longer is after this upgrade. Similar to the other deps we added here in the React17 upgrade, instead of doing a full audit was just added here.